jquery-sortable-lists.js

```diff
@@ -43,6 +43,7 @@
 				},
 				opener: {
 					active: false,
+					as: 'img',
```

Comment:
We can leave it without as. Only if user want to switch to html he set it to "html". May be that param could be named as as html: true,
Later in code(line 134) I would remove that condition. Next two conditions should be enough.